### PR TITLE
fix(native): Suggest release build

### PR DESCRIPTION
**Pull request by bruno-garcia**
On Windows it builds in Debug mode by default.

See also:

#439
#442

### CODE REVIEW

**Comment by jan-auer** [comment]
Re-assigning this. @Swatinem there's probably more we could be updating in the documentation from the readme. Particularly, builds, backends and configuration options have extensive info in the README that should rather go in docs.

File: src/includes/getting-started-install/native.mdx [hunk 1]
```diff
@@ -6,9 +6,9 @@ For example, `CMake` can be used like this (on macOS):
 
 ```shell
 # configure the cmake build into the `build` directory, with crashpad (on macOS)
-cmake -B build -D SENTRY_BACKEND=crashpad
+cmake -B build -D SENTRY_BACKEND=crashpad --config RelWithDebInfo 
 # build the project
-cmake --build build --parallel
+cmake --build build --config RelWithDebInfo --parallel
```
Comment (Swatinem) [inline src/includes/getting-started-install/native.mdx:11]: can you also re-generate the file tree? now that we install debug symbols properly? (or to check if we actually do)

Comment (bruno-garcia) [reply src/includes/getting-started-install/native.mdx:11]: Are you asking if the pdb gets copied over on install?

Comment (bruno-garcia) [reply src/includes/getting-started-install/native.mdx:11]: bump?

**Review by Swatinem — APPROVED** [review]

**Pull request merged**